src/_posts/databases/postgresql/2000-01-01-overview.md

```diff
@@ -0,0 +1,75 @@
+---
+title: Scalingo for PostgreSQL® Overview
+nav: Overview
+modified_at: 2023-11-20 00:00:00
+tags: database postgresql addon
+index: 1
+---
+
+## PostgreSQL®
+
+PostgreSQL® is a free, open-source, community-managed object-relational
+database management system (ORDBMS) focusing on SQL compliance and
+extensibility. It is well-known for its performances, reliability and
+robustness, making it a very powerful database, able to hanlde small to huge
+workloads.
+
+Besides the core features (ACID transactions, views, triggers, foreign keys,
+...) PostgreSQL® also offers solutions for more specialized usages, such as
+object storage with the native `jsonb` format, or for specific dataset kinds,
+such as geographic datasets with the PostGIS extension or for time-series
+datasets with TimescaleDB.
+
+
+## Scalingo for PostgreSQL®
+
+Scalingo for PostgreSQL® is our official PostgreSQL® addon. [We have several
+plans available](https://scalingo.com/databases/postgresql#database-compare),
+which you can chose of depending on your needs and budget. As always, a free
+plan is available for you to test our offer during a limited amount of time.
+
+### Features
+
+All our plans are fully managed. They allow you to enable the available
+extensions, and allow you to scale your database to another plan.
+
+They also include access to metrics and logs, allowing for quick and easy
+monitoring and auditing. For more information regarding this topic, please [see
+our dedicated page]({% post_url databases/postgresql/2000-01-01-monitoring %}).
+
+From the security point of view, they all come with a TLS connection and
+encryption at rest.
+
+In addition to the above, Starter plans include daily on-demand backups, as
+well as 7-days Point-in-time Recovery (PITR) backups. All backups are encrypted
+and stored externally. For more information regarding this topic, please
+[see our dedicated page]({% post_url databases/postgresql/2000-01-01-backing-up %}).
+
+For the most demanding environments, our Business plans include a fully-managed
+cluster made of 2 PostgreSQL® nodes (one primary and one replica) and 2 HAProxy
+(one primary and one backup) as entrypoint to your cluster private network.
+The communication between the PostgreSQL® nodes is encrypted. Chosing such a
+plan drastically lowers the risk of unavailability, be it during a maintenance
+operation (database upgrade, for example) or because of a platform issue.
+
+### Available Version
+
+The latest version of PostgreSQL® available is **`14.8.0-1`**.
```

Comment:
suggestion: can you update our internal procedure to update this version number when it changes?

Reply:
Sure :+1: (well, it's done :))